docs/platform-setup.md

```diff
@@ -18,3 +18,124 @@ For each job, disable the GitHub multibranch status:
 - Select "Disable GitHub Notifications".
 
 - Click on "Save".
+
+## Deploy Nexus Docker Registry
+
+### Prerequisite
+
+- Nexus is installed.
+- NGINX Ingress Controller is installed.
+
+Deploying docker registry with nexus should follow this steps:
+
+### Create New Docker Repository
+
+Open the platform team's [Nexus](https://nexus.platform.dev.nuxeo.com/). Start new session using admin user
+
+#### Blop Storage And Repository Configuration
+
+- Create a blob storage : `docker-registry`
+
+  - In Server Administration and Configuration > Blob Stores,  click on the "Create blob store" button
+
+  - Complete all required fields and click on "Save"
+
+- Create a docker (hosted) nexus repository: `docker-registry`
+
+  - In Server Administration and Configuration > Repositories,  click on the "Create repository" button
+
+  - Complete all required fields and click on "Save"
+
+### Cleanup Policy Configuration
+
+For each nexus docker registry, create a new policy cleanup and  apply to it.
+
+#### Create New Cleanup Policy
+
+- In Server Administration and Configuration > Cleanup Policies,  click on the "Create Cleanup Policy" button
+
+- To keep all nuxeo builder and jenkins master images ( latest tag for example ) add this asset name matcher `~(.*nuxeo/builder-.*/manifests/latest|.*nuxeo/platform.*-jenkinsx/manifests/[0-9]{1,}.[0-9]{1,}.[0-9]{1,})`
+
+- Complete all required fields and click on "Save"
+
+#### Apply Cleanup Policy
+
+- Open an existing nexus repository configuration for instance [docker-registry](https://nexus.platform.dev.nuxeo.com/#admin/repository/repositories:docker-registry)
+
+- Go done to cleanup section and select the corresponding item
+
+- Click on "Save".
+
+### Create New Docker Ingress Service
+
+This ingress service will act as Reverse Proxy for nexus docker repository.
+The reverse proxy would direct docker commands ( pull, push or login ) received at
+
+`https://docker.platform.dev.nuxeo.com`
```

Comment:
Currently this is `https://docker-registry.platform.dev.nuxeo.com`, shouldn't we want to use the same?

Reply:
`docker-registry` refers to the [Docker registry Helm chart](https://github.com/helm/charts/tree/master/stable/docker-registry) installed by default with the Jenkins X platform.
Since we're disabling it, it doesn't seem a problem to me to use `docker.xxx` directly.
Just like `docker.packages.nuxeo.com`.

Reply:
Also, maybe remove the backticks: though it will add a Markdown linter warning, we might want those URLs' to be clickable.